src/util/system.cpp

```diff
@@ -420,7 +420,7 @@ std::optional<unsigned int> ArgsManager::GetArgFlags(const std::string& name) co
     return std::nullopt;
 }
 
-const fs::path& ArgsManager::GetBlocksDirPath() const
+const fs::path ArgsManager::GetBlocksDirPath() const
```

Comment:
b80d0ab957655d28bbe495c421c147e3e601fbf6: backport 29040

Reply:
29040 has too many extra changes that are early to do, should I do as backported?

Reply:
I mean, maybe rename it to "partial merge 29040"?